tests/tck/utils/assertions.py

```diff
@@ -1,10 +1,76 @@
+from numbers import Number
+from redisgraph import Node, Edge
 def is_number_tryexcept(s):
-    """ Returns True is string is a number. """
+    """ Returns True if string is a number. """
+    if not isinstance(s, (Number, basestring)):
+        return False
     try:
         float(s)
         return True
     except ValueError:
         return False
+    
+def prepareActualValue(actualValue):
+        actualValue = prepareString(actualValue)
+        actualValue = stringToNumeric(actualValue)
+        actualValue = nodeToString(actualValue)
+        actualValue = edgeToString(actualValue)
+        return actualValue
+
+def prepareExpectedValue(expectedValue):
+    # string preparation
+    expectedValue = prepareString(expectedValue)
+    if expectedValue == "true":
+        expectedValue = True
+    if expectedValue == "false":
+        expectedValue = False
+    if expectedValue == "null":
+        expectedValue = None
+    expectedValue = stringToNumeric(expectedValue)
+    return expectedValue
+
+def prepareString(value):
+    if isinstance(value, basestring):
+        value = value.replace("'", "")
+        value = value.replace('"', "")
+    return value
+
+def stringToNumeric(value):
+    if is_number_tryexcept(value):
+        value = float(value)
+        if value.is_integer():
+            value = int(value)
+    return value
+
+def nodeToString(value):
+    if isinstance(value, Node):
+        res = '('
+        if value.alias:
+            res += value.alias
+        if value.label:
+            res += ':' + value.label
+        if value.properties:
+            props = ','.join(key+': '+str(val) for key, val in value.properties.items())
+            if value.label:
+                res+=" "
+            res += '{' + props + '}'
+        res += ')'
+        value = res
+    return value
+
+def edgeToString(value):
```

Comment:
what if value is not an Edge object the function will return the original value.

Reply:
fixed